ui/src/app/subnets/views/SubnetDetails/StaticRoutes/StaticRoutes.tsx

```diff
@@ -151,9 +152,9 @@ const StaticRoutes = ({ subnetId }: Props): JSX.Element | null => {
   );
   const subnets = useSelector(subnetSelectors.all);
   const subnetsLoading = useSelector(subnetSelectors.loading);
-  const isSuperUser = useSelector(userSelectors.isSuperUser);
+  const isAdmin = useSelector(authSelectors.isAdmin);
```

Comment:
thanks for updating @huwshimi 

I was looking in the `user` selectors looking for priviledges as the `auth` is in the user slice.

As a side note, It feels like they should live in a single place, we're already manipulating "auth" in `user`, e.g.:
https://github.com/canonical-web-and-design/maas-ui/blob/master/ui/src/app/store/user/slice.ts#L77-L79